common/src/api/external/mod.rs

```diff
@@ -629,12 +629,19 @@ pub struct IdentityMetadataUpdateParams {
 )]
 #[serde(rename_all = "snake_case")]
 pub enum InstanceState {
-    Creating, // TODO-polish: paper over Creating in the API with Starting?
+    /// An Instance record exists but the Instance is currently being created on the Sled.
```

Comment:
I don't think this would be clear to the API consumer; I would be happy to think about and suggest other language if that would be helpful.

This is a lot of states that an instance can be in. Are they all meaningfully different to the user? (i.e. that TODO-polish before this change)